docs/UserGuide.md

```diff
@@ -107,6 +107,7 @@ The `Commits Panel` allows users to see the commits attributed to a specific aut
 * The date range for the `Chart Panel` can be updated by clicking on the "Show ramp chart for this period" below the name of the author.
 * The ramp slices displayed in the ramp chart for the `Commits Panel` represents individual commits.
 * The commit messages body can be expanded or collapsed by clicking on the `...` icon beside each commit message title.
+* To promote Git's 50/72 commit message rule, the commit message title shows a dotted border at 50 characters while the commit message body shows a dotted border at 72 characters.
```

Comment:
I think something like this may be better

> To promote and encourage the 50/72 rule for commit messages, a dotted border will be shown for commit message subject that exceeds 50 characters and shown for commit message body after the 72nd character mark.

or

> * To promote and encourage the 50/72 rule for commit messages, a dotted border will be shown for:
>   * commit message subject that exceeds 50 characters 
>   * commit message body after the 72nd character mark.

Reply:
thanks for the advice, have went with the second option

Reply:
Better to make them consistent?